jetstream/experimenter.py

```diff
@@ -118,7 +121,7 @@ class ExperimentV4:
     active: bool
     features: List[str]
     branches: List[Branch]
-    startDate: dt.datetime
+    startDate: Optional[dt.datetime]
```

Comment:
Looks like the specs got updated at some point. Updated `startDate` here to reflect that.